### PR TITLE
2.0.0

### PR DESCRIPTION
**Pull request by jjasghar**
* Updated test-kitchen to use chef_zero
* Changed Berksfile to be `source`
* Updated location for the `chef-full.erb` bootstrap template
* Updated the server_test minitest to look for apache, added serverspec for in.tftp
* Created `examples/pxe_dust/` to have databag for integration testing
* Updated `default.json` from the above for 14.04
* Added serverspec tests for integration testing
* Other random updates for 14.04 updates
* Removed 10.04/13.04 support
* Added dhcpd server recipe

### CODE REVIEW

File: test/integration/common/serverspec/spec_helper.rb [hunk 1]
```diff
@@ -0,0 +1,3 @@
+require 'serverspec'
```
Comment (ryancragun) [inline test/integration/common/serverspec/spec_helper.rb]: Y U NO INSPEC?

**Comment by ryancragun** [comment]
:rocket:

My only gripes are no inspec and there's a lot of mixed colons and semi-colons.  Maybe give it the ole `rubocop -a`.

**Comment by adamleff** [comment]
Echoing @ryancragun's comments - :+1: otherwise.

File: test/integration/common/serverspec/spec_helper.rb (hunk 1 above)
Comment (jjasghar) [inline test/integration/common/serverspec/spec_helper.rb]: Yeah yeah. I'll updated it to inspec at a later date.  It will happen just not right now.

**Pull request merged**